test/cppsuite/configs/config_example_test_default.txt

```diff
@@ -29,7 +29,15 @@ workload_generator=
     ),
     read_threads=1,
     update_threads=1,
-    value_size=10
+    value_size=10,
+    update_config=
+    (
+        ops_per_second=1
```

Comment:
Can you check if this can be removed as the default value is 1 ?

Reply:
Let's do that in the configuration default expansion ticket.

Reply:
WT-7296